.editorconfig

```diff
@@ -13,18 +13,18 @@ charset = utf-8
 
 # Indentation override for all JS
 [**.js]
-indent_style = space
+indent_style = tab
```

Comment:
I would personally use `spaces`

Reply:
is this evergreen bikeshed discussion on every project 😅

let's go with tabs :D

Reply:
Because she told you that she needs some `space`

I would not mind to use tabs but the code is not properly formatted
<img width="1646" alt="Screenshot 2021-09-29 at 13 38 23" src="https://user-images.githubusercontent.com/22471030/135261374-366f5cb6-451e-4d9e-b4f0-9bcda8550053.png">

Reply:
Let's agree to use "space: 2", I'm currently fighting to fix "Mixed spaces and tabs" on .vue files.

Reply:
well, tabs could be space 2 tho :)
then there is another discussion about indentation... 2 or 4 😄